sherpa/io.py

```diff
@@ -259,18 +259,18 @@ def read_data(filename, ncols=2, colkeys=None, sep=' ', dstype=Data1D,
        The name of the ASCII file to read in.
     ncols : int, optional
        The number of columns to read in (the first ``ncols`` columns
-       in the file). This is ignored if ``colkeys`` is given.
+       in the file). This is ignored if `colkeys` is given.
```

Comment:
Not for this PR, but we need to decide whether we want to use ``ncols`` or `colkeys` (and it's probably something we may want to normally do one way but allow to change).

I currently don't have a preference.